docs/conch/examples/sshsimpleserver.py

```diff
@@ -217,22 +217,20 @@ class ExampleFactory(factory.SSHFactory):
         b'ssh-connection': connection.SSHConnection
     }
 
+    def __init__(self):
```

Comment:
I agree with [@cdunklau's comment](https://github.com/twisted/twisted/pull/1096#issuecomment-457143556) - the constructor should accept these arguments.  That can be done in a separate PR, though.